debian/control

```diff
@@ -946,142 +946,150 @@ Description: IJ Printer Driver for Linux.
+	cnijfilter-e560series,
+	cnijfilter-e600series,
+	cnijfilter-e610series,
+	cnijfilter-ip100series,
+	cnijfilter-ip2800series,
 	cnijfilter-ip4800series,
-	cnijfilter-mg6100series,
-	cnijfilter-mg8100series,
-	cnijfilter-mx360series,
-	cnijfilter-mx410series,
-	cnijfilter-mx420series,
-	cnijfilter-mx880series,
+	cnijfilter-ip4900series,
+	cnijfilter-ip7200series,
+	cnijfilter-ip8700series,
 	cnijfilter-ix6500series,
+	cnijfilter-ix6700series,
+	cnijfilter-ix6800series,
 	cnijfilter-mg2100series,
+	cnijfilter-mg2200series,
+	cnijfilter-mg2400series,
+	cnijfilter-mg2500series,
 	cnijfilter-mg3100series,
+	cnijfilter-mg3200series,
+	cnijfilter-mg3500series,
 	cnijfilter-mg4100series,
+	cnijfilter-mg4200series,
+	cnijfilter-mg5100series,
+	cnijfilter-mg5200series,
 	cnijfilter-mg5300series,
+	cnijfilter-mg5400series,
+	cnijfilter-mg5500series,
+	cnijfilter-mg6100series,
 	cnijfilter-mg6200series,
+	cnijfilter-mg6300series,
+	cnijfilter-mg6400series,
+	cnijfilter-mg6500series,
+	cnijfilter-mg7100series,
+	cnijfilter-mg8100series,
 	cnijfilter-mg8200series,
-	cnijfilter-ip4900series,
-	cnijfilter-e500series,
-	cnijfilter-mx710series,
-	cnijfilter-mx890series,
-	cnijfilter-mx370series,
-	cnijfilter-mx430series,
-	cnijfilter-mx510series,
-	cnijfilter-e600series,
 	cnijfilter-mp230series,
-	cnijfilter-mg2200series,
-	cnijfilter-e510series,
-	cnijfilter-mg3200series,
-	cnijfilter-mg4200series,
-	cnijfilter-ip7200series,
-	cnijfilter-mg5400series,
-	cnijfilter-mg6300series,
-	cnijfilter-mx720series,
-	cnijfilter-mx920series,
+	cnijfilter-mp250series,
+	cnijfilter-mp280series,
+	cnijfilter-mp495series,
+	cnijfilter-mx360series,
+	cnijfilter-mx370series,
 	cnijfilter-mx390series,
+	cnijfilter-mx410series,
+	cnijfilter-mx420series,
+	cnijfilter-mx430series,
 	cnijfilter-mx450series,
-	cnijfilter-mx520series,
-	cnijfilter-e610series,
-	cnijfilter-mg7100series,
-	cnijfilter-mg6500series,
-	cnijfilter-mg6400series,
-	cnijfilter-mg5500series,
-	cnijfilter-mg3500series,
-	cnijfilter-mg2400series,
-	cnijfilter-mg2500series,
-	cnijfilter-p200series,
-	cnijfilter-ix6700series,
-	cnijfilter-ix6800series,
-	cnijfilter-ip2800series,
 	cnijfilter-mx470series,
+	cnijfilter-mx510series,
+	cnijfilter-mx520series,
 	cnijfilter-mx530series,
-	cnijfilter-ip8700series,
-	cnijfilter-e560series,
-	cnijfilter-e400series,
-	cnijfilter-i550series,
-	cnijfilter-i850series,
-	cnijfilter-i950series,
-	cnijfilter-pixus250iseries,
-	cnijfilter-pixus255iseries,
-	cnijfilter-i560series,
-	cnijfilter-i860series,
-	cnijfilter-i990series,
-	cnijfilter-ip3000series,
-	cnijfilter-bjs700series,
-	cnijfilter-mp750series,
-	cnijfilter-mp780series,
-	cnijfilter-ip4000series,
-	cnijfilter-ip5000series,
-	cnijfilter-ip8500series,
-	cnijfilter-ip5200series,
-	cnijfilter-ip5200rseries,
-	cnijfilter-mp620series
+	cnijfilter-mx710series,
+	cnijfilter-mx720series,
+	cnijfilter-mx880series,
+	cnijfilter-mx890series,
+	cnijfilter-mx920series,
+	cnijfilter-p200series
+Recommends: printer-driver-cnijfilter-32
```

Comment:
I forgot how this crazy package puts things together, but I think it would be best to just mark the i386 only drivers with `[i386]` rather than using Recommends.